pkgs/by-name/ap/apple-sdk/common/propagate-inputs.nix

```diff
@@ -66,8 +64,11 @@ self: super: {
   buildPhase =
     super.buildPhase or ""
     + ''
-      for header in '${lib.getDev libiconv}/include/'* '${lib.getDev ncurses}/include/'*; do
+      for header in '${lib.getDev libiconv}/include/'* '${lib.getDev ncurses}/include/'* '${cupsHeaders}/include/'*; do
         ln -s "$header" "usr/include/$(basename "$header")"
       done
     '';
+
+  # Exported to allow the headers to pass the requisites check in the stdenv bootstrap.
+  passthru = (super.passthru or { }) // { cups-headers = cupsHeaders; };
```

Comment:
Any reason for the different case here?

Reply:
The convention for packages is kebab case, but the convention for variables is camel casing. This is an exported package, so I followed the package convention. I could change the variable name, but it would make the diff bigger.